HelpCentre/Prod/flows/intro-to-helpcentre.md

```diff
@@ -0,0 +1,1229 @@
+
+## Global variables
+
+The following variable(s) are set in the `settings` global dictionary
+
+* `contentrepo_qa_token` used to auth api calls to CMS
+* `working_hours_start_day`the day of the week the helpdesk opens (note days of the week starts at Sunday being 1, and Saturday being 7)
+* `working_hours_end_day` the day of the week the helpdesk closes
+* `working_hours_start_hour` The hour that the helpdesk opens
+* `working_hours_end_hour`The hour that the helpdesk closes
+* `mc_ndoh_hub_token` used to authenticate calls to AAQ via MomConnect NDOH hub
+* `turn_qa_token` used to authenticate calls to the Turn instance
+
+## Content dependencies
+
+* `mnch_onboarding_error_handling_button`
+* `plat_help_welcome_help_centre_first`
+* `plat_help_welcome_help_centre_returning`
+* `plat_help_medical_emergency`
+* `plat_help_emergency_contact_numbers`
+* `plat_help_search_myhealth_prompt`
+* `plat_help_technical_issue_prompt`
+* `plat_help_invalid_media_catch_all`
+* `plat_help_general_catch_all`
+* `plat_help_medical_emergency_secondary`
+* `plat_help_faqs_topics_list`
+* `plat_help_faq_topic_content`
+* `plat_help_acknowledgement_positive_`
+* `plat_help_acknowledgement_negative_`
+* `plat_help_help_desk_entry_offline`
+* `plat_help_acknowledgement_negative_`
+
+```stack
+trigger(on: "MESSAGE RECEIVED") when has_only_phrase(event.message.text.body, "i2h")
+# interaction_timeout(60)
+
+```
+
+# FetchError
+
+```stack
+card FetchError, then: InitHelpdesk do
+  # Fetch and store the error message, so that we don't need to do it for every error card
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "mnch_onboarding_error_handling_button"]
+      ],
+      headers: [["Authorization", "Token @global.settings.contentrepo_qa_token"]]
+    )
+
+  # We get the page ID and construct the URL, instead of using the `detail_url` directly, because we need the URL parameter for `get` to start with `https://`, otherwise stacks gives us an error
+  page_id = search.body.results[0].id
+
+  page =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @global.settings.contentrepo_qa_token"]]
+    )
+
+  button_error_text = page.body.body.text.value.message
+end
+
+```
+
+# Initialise Helpdesk
+
+```stack
+card InitHelpdesk, then: SetWorkingHours do
+  current_date = now()
+  # Add two hours for SA Time
+  current_date = datetime_add(current_date, 2, "h")
+
+  current_day = weekday(current_date)
+  current_hour = hour(current_date)
+  log("Current hour = @current_hour")
+  opening_day = global.settings.working_hours_start_day
+  closing_day = global.settings.working_hours_end_day
+end
+
+# Weekdays starts on Sunday(1) and ends on Saturday(7)
+card SetWorkingHours when current_day < opening_day or current_day > closing_day,
```

Comment:
I think we need to be careful with this, since these are configurable. If we have something like a helpdesk that's open Tuesday to Sunday, then opening day is 3 and closing day is 1, but then this logic doesn't work

Reply:
Thanks, I agree, will discuss this further on Slack